patch-package
--- a/node_modules/react-dom/cjs/react-dom.development.js
+++ b/node_modules/react-dom/cjs/react-dom.development.js
@@ -8643,7 +8643,7 @@ var enableUserTimingAPI = true;
 // render phase
 var enableGetDerivedStateFromCatch = false;
 // Suspense
-var enableSuspense = false;
+var enableSuspense = true;
 // Helps identify side effects in begin-phase lifecycle hooks and setState reducers:
 var debugRenderPhaseSideEffects = false;
 
